jenkins-integration/beaker/gatling/10_start_gatling_scenario.rb

```diff
@@ -0,0 +1,35 @@
+test_name "Start gatling scenario"
+
+simulation_id = ENV['GATLING_SIMULATION_ID']
+gatling_scenario = ENV['GATLING_SCENARIO']
+sut_hostname = ENV['GATLING_SUT_HOSTNAME']
+
+if not simulation_id
+  abort "GATLING_SIMULATION_ID environment variable required"
+elsif not gatling_scenario
+  abort "GATLING_SCENARIO environment variable required"
+elsif not sut_hostname
+  abort "GATLING_SUT_HOSTNAME environment variable required"
+end
+
+repo_url = "https://github.com/puppetlabs/gatling-puppet-load-test.git"
+repo_path = "/root/gatling-puppet-load-test_#{simulation_id}"
+
+sbt_path = "#{repo_path}/simulation-runner"
+
+# Sets the environment variables and runs sbt
+sbt_command = "PUPPET_GATLING_SIMULATION_CONFIG=config/scenarios/#{gatling_scenario} PUPPET_GATLING_SIMULATION_ID=#{simulation_id} PUPPET_GATLING_MASTER_BASE_URL=#{sut_hostname} sbt run"
+
+# To keep log/result files separate on the gatling machine
+step "Clone gatling-puppet-load-test into new folder for this scenario"
+on(gatling, "git clone #{repo_url} #{repo_path}")
+
+step "Start gatling with scenario on gatling machine"
+# This seems like an ugly but accepted way of starting background tasks over ssh
+nohup_cmd = "nohup bash -c 'cd #{sbt_path} && #{sbt_command}' &> #{sbt_path}/sbt.out < /dev/null &"
+on(gatling, nohup_cmd)
```

Comment:
So, with this approach, we wouldn't have any way of knowing when the run actually completed, correct?  In other words, the jenkins job would end immediately and would leave the rest of the stuff running indefinitely?

I'm thinking we really want to have the jenkins job keep running if we can, so that it can collect artifacts from the run when it completes.  I believe that if you look at the old code, we were doing this by using the jenkins `sbt` plugin:

https://wiki.jenkins-ci.org/display/JENKINS/sbt+plugin

Then we were using the `puppet-gatling-jenkins-plugin` after that, to collect the data and build graphs that would show up in jenkins itself.

So maybe we don't really need to have gatling represented in the beaker config at all?  gatling would end up being more of a jenkins-side requirement.  And for simple scenarios, you'd just run gatling directly on the master jenkins node, but in production, we could have some jenkins slaves that were especially for gatling runs, and restrict the gatling jobs to only run on those?  e.g. maybe we eventually set up one of those 32-core blades as a Jenkins slave?